handle/Timer.lua

```diff
@@ -1,30 +1,190 @@
 require('CeresStdLib.base.Native')
 require('CeresStdLib.base.Log')
-local free_timers = {}
+
+local Timer 		= {}
+local free_timers	= {}
```

Comment:
I never heard that native timer functions fail. Could you evaluate on such cases a little, maybe provide failing test case?

Reply:
Issues with Repeating Timers failing to repeat once they are resumed (after a pause).
There's also the ResumeTimer thing which can crash the game if called before TimerStart was invoked for the timer object.
There was a thread in the Lab section of Hive delving into these things.
https://www.hiveworkshop.com/threads/issues-with-timer-functions.309433/

The Timer.lua was overwritten to address at least those two above.